docroot/profiles/custom/cgov_site/modules/custom/cgov_image/config/install/image.style.video_display_large_medium_small_full_content.yml

```diff
@@ -0,0 +1,25 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - crop.type.4x3
+  module:
+    - crop
+name: video_display_large_medium_small_full_content
+label: 'Video Display: Large/Medium/Small & Full Content'
```

Comment:
What is this style? There should be a style for each:
* Full
* Large
* Medium
* Small

Reply:
Firstly, Full and Large end up being the same size on screen - 994 px. So the "Full" display mode for a video should use the "large" image mode. (Full will go away at some point anyway)

As far as Large, Medium and Small:

At 0-1023 pixels wide we are full-width, which ends up being 991px in non-retina. If we go by 2x then we get 1982, which is not 1988. 
On desktop and up, the image is 994, which gives us 1988, which is what is in requirements.

At the very least there should be a different Large vs Medium/Small where Medium/Small is resized to 1982. 

**HOWEVER**, part of the use of image styles is future responsive images. This is where we render an image with multiple sizes, which multiple max sizes. In this case, is important to separate all 3 image styles. 

So while they have the same max-width, they are 3 different display cases, and if we turned on responsive images, which will happen some day, we would end up having to split out all the sizes.

So for the implementation, let's create 3 separate "Video Display: XXX" image styles for Large, Medium and Small.

Reply:
@navsunka I updated the requirements on the ticket so it broke out "Video Display: Full Content", "Video Display: Large", "Video Display: Medium", "Video Display: Small" to account for that part of Bryan's comment

Reply:
However, as I noted, there does not need to be "Video Display: Full Content"
> Firstly, Full and Large end up being the same size on screen - 994 px. So the "Full" display mode for a video should use the "large" image mode. (Full will go away at some point anyway)

@navsunka  can you please remove the Full image style and just point Full at the large image style.

Reply:
Removed the Full image style.